examples/dc/plot_dc_schlumberger_array.py

```diff
@@ -0,0 +1,268 @@
+"""
```

Comment:
This one I experimented with their "notebook" like format. It might be better to use a different plugin for these larger walk throughs...